save-common/src/commonMain/kotlin/org/cqfn/save/entities/ProjectDto.kt

```diff
@@ -0,0 +1,16 @@
+package org.cqfn.save.entities
+
+import kotlinx.serialization.Serializable
+import org.cqfn.save.repository.GitRepository
+
+/**
+ * Data class of information about project
+ *
+ * @property project project
+ * @property gitRepository github repository
+ */
+@Serializable
+data class ProjectDto(
```

Comment:
Let's call it `ExecutionRequest` or something similar. In future it should support execution on standard suites as well as git repos.